### ISSUE TITLE
Boundary Edges for Open meshes only?

### ISSUE THREAD

**Issue by saravanan2020** [gradientspace/geometry3Sharp#50]
I heard boundary edges can be extracted for open meshes only. That means for Non-manifold meshes only? Is it true?
Please clear me.
Thanks.

**Comment by rms80** [comment]
yes, only open meshes have a boundary to be extracted. But open meshes are not non-manifold, at least as I would define it. Non-manifold means more than 2 triangles connected at an edge. This is not possible to represent with DMesh3, as it's data structures only can store 2 triangles per edge. 

I guess technically it is "manifold-with-boundary".
